Solution4.py

```diff
@@ -0,0 +1,21 @@
+# Question 4 Solution
+import csv
+header=['sid','name','salary']
+rows=[[1,'krishna',4600],[2,'om',4000],[2,'ram',4000],[3,'radha',4600],[5,'sai',4200]]
+temp=0     #counter
+
+# Writing file
+with open("C:\sqlite3\salary.csv","w",newline="") as write_obj:
```

Comment:
why write newline=''?

Reply:
without newline between each records a empty list will be automatically inserted which can cause problem while using if statements to check conditions